android/Staccato_AN/app/src/main/java/com/on/staccato/presentation/memorycreation/viewmodel/MemoryCreationViewModel.kt

```diff
@@ -114,13 +102,58 @@ class MemoryCreationViewModel
             }
         }
 
+        private fun createThumbnail(
+            uri: Uri?,
+            url: String? = null,
+        ) {
+            val isEqualUri: Boolean? = _thumbnail.value?.isEqualUri(uri)
+            if (isEqualUri == false) {
+                thumbnailJobs[_thumbnail.value?.uri]?.cancel()
+                _thumbnail.value = ThumbnailUiModel(uri = uri, url = url)
```

Comment:
덧붙여서, `private fun createThumbnail()`의 url 매개 변수는 항상 null이 되겠네요.

```kotlin
        fun createThumbnailUrl(
            context: Context,
            uri: Uri,
        ) {
            _isPhotoPosting.value = true
            createThumbnail(uri) // uri만 넣어주고 있음. url은 항상 default인 null
            registerThumbnailJob(context, uri)
        }
```

그렇다면 url 매개변수를 제거하고 아래처럼 null을 직접 넣어주는 것이 더 명시적일 것 같아요~!

```suggestion
                _thumbnail.value = ThumbnailUiModel(uri = uri, url = null)
```

Reply:
헉 그렇네요! 수정했습니다~

`반영 커밋` : [382d063](https://github.com/woowacourse-teams/2024-staccato/pull/542/commits/382d06376d8844a1d078fc93ea8986541bce9276)